src/Sentry/Laravel/Tracing/ServiceProvider.php

```diff
@@ -36,6 +36,10 @@ public function boot(): void
     public function register(): void
     {
         $this->app->singleton(Middleware::class);
+
+        $this->app->booted(function () {
+            $this->app->make(Middleware::class)->setBootedTimestamp(microtime(true));
+        });
```

Comment:
So this won't work with Lumen since it doesn't have a `booted` callback...

I guess we could do something like this for Lumen and just accept there is no booting span.

```suggestion
		if (!$this->app instanceof Lumen) {
	        $this->app->booted(function () {
    	        $this->app->make(Middleware::class)->setBootedTimestamp(microtime(true));
        	});
        }
```

Reply:
This is a very good point. I did not think about `Lumen` - we had this check in `Middleware` and I should have moved it into the service provider. With this change we also make it _easier_ for Lumen to enable this metric - they just can invoke `setBootedTimestamp()` from whatever routine is appropriate and `Middleware` will do the rest.